### ISSUE TITLE
Please provide performance states

### ISSUE THREAD

**Issue by akshaychauhan7737** [sominator/web-api-project#1]
I have read the article. https://www.freecodecamp.org/news/i-built-a-web-api-with-express-flask-aspnet/amp/
The most important thing that you should tell was how they performed. You should test them for there time to response, how much request each server can handel.

**Comment by sominator** [comment]
> I have read the article. https://www.freecodecamp.org/news/i-built-a-web-api-with-express-flask-aspnet/amp/
> The most important thing that you should tell was how they performed. You should test them for there time to response, how much request each server can handel.

Thanks for the suggestion!

**Comment by AJenbo** [comment]
I agree that performance stats are nice, but depending on how you measure and what your target is, the numbers can be missleading. Especially at a stage where no optimization has taken place. 

From my perspective, and I think also the article, the most valuable resource is the human effort required to develop and maintain the code. 

Performed wise I think it should more often then not be reduced to a) can it scale, b) what are the options for scaling. 

With the vastly different database engines selected in each implementation the numbers are probably going to be heavily skewed to the database engines characteristics.

Still it might be fun to see numbers for through put, latency, concurrency and latency under load 🙂

P.s. Would you be interested in PRs for other backend implementations?
(could you also share the postman collection)

**Comment by sominator** [comment]
> From my perspective, and I think also the article, the most valuable resource is the human effort required to develop and maintain the code.

This is exactly what I was going for with the article!  I've spent a lot of time researching those three and thought it might save time for others intending to do the same thing. 
 
> P.s. Would you be interested in PRs for other backend implementations?
> (could you also share the postman collection)

I'd love to see them.  The Postman collection is _really_ simple: https://www.getpostman.com/collections/3ad8d51125d6c3763efc

**Comment by AJenbo** [comment]
This doesn't appear to have been updated as it points to this url http://localhost:3000/subscribers

**Comment by sominator** [comment]
> This doesn't appear to have been updated as it points to this url http://localhost:3000/subscribers

That's weird; looks like it didn't save properly.  Here it is again: https://www.getpostman.com/collections/3ad8d51125d6c3763efc
